wonkyung/.gitignore

```
@@ -0,0 +1,4 @@
+PORT: insert your PORT number
+
+
+you have to make your own schema file
```

해당 부분에 `.env.sample` 파일에 있는 내용 두가지가 추가되어야 할 것 같습니다.

추가로 db 폴더 내부의 `schema.sql` 파일이 github 상에서 공유되지 않도록 해당 파일에도 추가해주세요!
이미 추가가 되어있는데, `git rm --cached "file-path"` 명령어를 이용하여 삭제 후 다시 add ~ commit ~ push 해주세요!

```suggestion
.env
node_modules/
db/schema.sql
```